### ISSUE TITLE
A11y_AzureDataStudioAzureArcExtension_AzureArcController_ConnecttoExistingAzureArcController_ScreenReader: NVDA does not announces the change in the notification when the default path is deleted from "Kube config file path" editbox

### ISSUE THREAD

**Issue by Shivamd1** [microsoft/azuredatastudio#15735]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#SH_AzureDataStudio_AzureArcExtension_Jun2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#NVDA;#WCAG4.2.12;#AINotInScope;

### Environment Details:
Application Name: Azure Data Studio -Azure Arc Extension
Windows Version: Win10
NVDA Version: 2020.4, Jaws(2020.2008.24)

### Repro Steps:

1. Launch the application-Azure Data Studio-Insiders and install extension Azure arc and Azure data ClI
2. Connect to Azure Arc Controller
3. Tab till Azure Arc Controller tree and tab to "Create new Azure Arc Controller"
4. Tab to "Kube config file path" edit box start nvda and delete the default path location and observe announcement.

### Actual:
NVDA does not announces the change in the notification when the default path is deleted from "Kube config file path" edit box.
The notification changes from "ENOENT: no such file or directory open, "path name" to ""ENOENT: no such file or directory, open"

### Expected:
NVDA should be announcing the change in the notification as well as the original notification.

### User Impact:
Screen Reader users will not get the proper information about change in notification.

**Comment by Shivamd1** [comment]
Closing as not repro

![image](https://user-images.githubusercontent.com/60880975/150948145-1339b6f2-7cb9-4c9f-9297-1a2d152adb4f.png)
